src/styles/_catalog.scss

```diff
@@ -14,6 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    cursor: pointer;
+    transition: 300ms;
+    transition-property: transform color;
```

Comment:
You can check how it works on dev tools
<img width="292" alt="Screenshot 2022-07-21 at 21 06 34" src="https://user-images.githubusercontent.com/47606379/180283463-4de3ef5a-ef36-4762-9512-8dc4367cc922.png">